doc/tutorials/plugins.md

```diff
@@ -441,6 +441,24 @@ Plugin *ELEKTRA_PLUGIN_EXPORT
 
 For further information see [the API documentation](https://doc.libelektra.org/api/current/html/group__plugin.html).
 
+### `elektraPluginGetGlobalKeySet`
+
+In order to enable communication between plugins which is more complex than what can be done with metadata, Elektra provides a global keyset which plugins can read from and modify.
+
+The keyset is initialized by the KDB and can be accessed by all plugins except for plugins created manually (e.g. with `elektraPluginOpen`).
```

Comment:
I'm not sure if it is important to mention that it is also closed with the KDB handle (`kdbClose`), and data is not shared with other KDB handles (i.e. it is not a mechanism for inter process communication).

Reply:
Yes, sounds also useful to me to mention this here.